src/app/EventManagement.cpp

```diff
@@ -124,6 +124,15 @@ void EventManagement::Init(Messaging::ExchangeManager * apExchangeManager, uint3
     mBytesWritten = 0;
 
     mMonotonicStartupTime = aMonotonicStartupTime;
+
+    if (apEventScheduler == nullptr)
```

Comment:
Is the goal to get rid of this case eventually?  If so, worth documenting that.

Reply:
added a comment.